app/tests/v1/test_meetup.py

```diff
@@ -7,41 +7,32 @@ class MeetupTestCase(BaseTestCase):
     '''Test definitions for a meetup'''
     def test_admin_create_meetup(self):
         '''Test an administrator can create a meetup'''
-        res = self.get_response_from_user_login(self.admin_registration, self.admin_login)
-        response_msg = json.loads(res.data.decode("UTF-8"))
-        access_token = response_msg["access_token"]
+        access_token = self.get_access_token(self.admin_registration, self.admin_login)
         res = self.client().post(
             '/api/v1/meetups',
             headers=self.get_authentication_headers(access_token),
             data=json.dumps(self.meetup)
         )
+        response_msg = json.loads(res.data.decode("UTF-8"))
         self.assertEqual(res.status_code, 201)
+        self.assertTrue(response_msg['data'])
```

Comment:
Assert the content of the data as well.

Reply:
I have now made changes by asserting that the data found in the meetups list is the one displayed by the response message.